data/json/monsters/zed_misc.json

```diff
@@ -367,6 +367,68 @@
     ],
     "armor": { "electric": 2 }
   },
+  {
+    "id": "mon_zombie_bruiser",
+    "type": "MONSTER",
+    "name": { "str": "bruiser zombie" },
+    "description": "This zombie's right arm has been grotesquely deformed into a massive mace-like shape.  What remains of its hand has been covered in spiky bone, and the upper arm's skin is clearly coming apart from the weight on it.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 2,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 110,
+    "speed": 80,
+    "material": [ "flesh", "bone" ],
+    "symbol": "Z",
+    "color": "red_white",
+    "scents_tracked": [ "sc_human", "sc_fetid" ],
```

Comment:
Most humaniod zombies outside of the predator line lack track by smell.  I notice every human zombie in this PR tracks by smell.  Is that intended?

Reply:
They're all copied from other zombies? The zombie brute right below this also tracks by scent. If this is a problem it's not one that I introduced, afaik.